package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
@TDA will submit a PR to the upstream **rwky/grunt-remarkable** repo so we don't have to dirty our package.json with **async** (or fork the repo and fix one dependency).